modules/wmi/charts.go

```diff
@@ -840,6 +854,24 @@ func addDimToCPUInterruptsChart(charts *Charts, coreID string) error {
 	return nil
 }
 
+func addDimToCPUFrequencyChart(charts *Charts, coreID string) error {
+	chart := charts.Get(cpuFrequencyChart.ID)
+	if chart == nil {
+		return fmt.Errorf("chart '%s' is not in charts", cpuFrequencyChart.ID)
+	}
+	dim := &Dim{
+		ID:   fmt.Sprintf("cpu_core_%s_frequency", coreID),
+		Name: "core" + coreID,
+		Algo: module.Incremental,
+		Div:  1000,
```

Comment:
```suggestion
	dim := &Dim{
		ID:   fmt.Sprintf("cpu_core_%s_frequency", coreID),
		Name: "core" + coreID,
```

- Div is not needed when stm:"frequency,1,1".
- the value is the gauge, Algorithm should be Absolute (the default when not set).